common/src/hash/cpp/Base64.cpp

```diff
@@ -136,7 +136,10 @@ int DecodeBase64(
   size_t buf = 0, len = 0;
 
   while (in < end) {
-    unsigned char c = kDecode[*in++];
+    // In base64 characters are guaranteed to be ASCII, and so are positive
+    // even if char is a signed type.
```

Comment:
There is no guarantee it is actually Base64. It is safe anyway because kDecode has 256 elements, but the comment is a bit mispointing. Consider rewording.

Reply:
Good catch, thanks! Reworded.